recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ requirements:
     - cuda-nsight 12.0.78
     - nsight-compute 2022.4.0.15
     - cuda-nvml-dev 12.0.76
-    - cuda-nvvp 12.0.90
+    - cuda-nvvp 12.0.90                 # [not aarch64]
```

Comment:
Since `cuda-nvvp` is [skipped on `linux_aarch64`]( https://github.com/conda-forge/cuda-nvvp-feedstock/blob/665682da90579570cdd2555f2dc0bf8ba5078367/recipe/meta.yaml#L22 ), drop this dependency on `linux_aarch64`. This should fix [the CI failure on `linux_aarch64`]( https://app.travis-ci.com/github/conda-forge/cuda-visual-tools-feedstock/jobs/611357971#L662 )

Reply:
I don't have commit permissions on this PR, but +1 on this.